### PR TITLE
magento/merchdocs#17: Remove links from headings in topics

### PR DESCRIPTION
**Pull request by atwixfirster**
## Purpose of this pull request

This pull request (PR) provides fixes for https://github.com/magento/merchdocs/issues/17

## Affected documentation pages

- https://docs.magento.com/m2/ce/user_guide/reports/sales-reports.html
- https://docs.magento.com/m2/ce/user_guide/reports/bi-essentials-report-builder.html

## Affected Magento editions

- [ ] Open Source
- [ ] Commerce
- [ ] B2B

## Links to Magento source code or PRs

- ...
- ...

## Additional information

Thank you!

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/merchdocs?pullRequest=32) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/merchdocs?pullRequest=32) before we can accept your contribution.<br/>

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/marketing/marketing-menu.md [hunk 1]
```diff
@@ -18,35 +18,35 @@ On the Admin sidebar, tap <span class="btn">Marketing</span>.
...
+Create [catalog]({{ site.baseurl }}{%- link marketing/price-rules-catalog.md -%}) and [cart]({{ site.baseurl }}{%- link marketing/price-rules-cart.md -%}) price rules that trigger discounts based on a variety of conditions. Set up [promotions]({{ site.baseurl }}{%- link marketing/promotions.md -%}) that spring into action when the required conditions are met.
 
 <!--{%- if "Default.EE-B2B" contains site.edition -%}-->
 Create [related product rules]({{ site.baseurl }}{%- link marketing/product-related-rules.md -%}) and manage [gift card accounts]({{ site.baseurl }}{%- link catalog/product-gift-card-accounts.md -%}).
 
-### [Private Sales]({{ site.baseurl }}{%- link marketing/promotions.md -%})
+### Private Sales
 
-Private sales and other catalog events are a great way to leverage your existing customer base to generate buzz and new leads with exclusive access for members only, or by invitation.
+[Private Sales]({{ site.baseurl }}{%- link marketing/promotions.md -%}) and other catalog events are a great way to leverage your existing customer base to generate buzz and new leads with exclusive access for members only, or by invitation.
```
Comment (shrielenee) [inline src/marketing/marketing-menu.md]: Revert this back to "Private sales"

File: src/marketing/marketing-menu.md [hunk 2]
```diff
@@ -18,35 +18,35 @@ On the Admin sidebar, tap <span class="btn">Marketing</span>.
...
-Customize all notifications sent from your store. Create [newsletters]({{ site.baseurl }}{%- link marketing/newsletters.md -%})
+Customize all [notifications]({{ site.baseurl }}{%- link marketing/communications.md -%}) sent from your store. Create [newsletters]({{ site.baseurl }}{%- link marketing/newsletters.md -%})
 and publish [RSS]({{ site.baseurl }}{%- link marketing/rss-feed.md -%}) feeds.
 
 <!--{%- if "Default.EE-B2B" contains site.edition -%}-->
 Set up rules that send [email reminders]({{ site.baseurl }}{%- link marketing/email-reminder-rules.md -%}) to customers whenever the conditions are met.
 
 <!--{% endif %}-->
-### [SEO &amp; Search]({{ site.baseurl }}{%- link marketing/seo-search.md -%})
+### SEO &amp; Search
```
Comment (shrielenee) [inline src/marketing/marketing-menu.md]:
```suggestion
### SEO and Search
```

File: src/marketing/rewards-loyalty.md [hunk 3]
```diff
@@ -5,21 +5,21 @@ title: Rewards & Loyalty
...
+Points can be [redeemed]({{ site.baseurl }}{%- link marketing/reward-exchange-rates.md -%}) by admin users and (if enabled) customers during checkout. In the Payment Method section, a Use my Reward Points checkbox appears above the enabled payment methods. The available points and monetary exchange rate is included. If the available balance is greater than the order grand total, no additional payment methods is required. The amount of reward points applied to the order appears with the order totals, subtracted from the grand total, similar to a store credit or gift cards. If reward points are used in conjunction with store credit or a gift card, the reward points are deducted first, and the store credit or gift card is deducted if the order total is greater than the redeemable amount of reward points.
```
Comment (shrielenee) [inline src/marketing/rewards-loyalty.md]:
```suggestion
Points can be [redeemed]({{ site.baseurl }}{%- link marketing/reward-exchange-rates.md -%}) by admin users and customers (if enabled) during checkout. In the Payment Method section, a Use my Reward Points checkbox appears above the enabled payment methods. The available points and monetary exchange rate is included. If the available balance is greater than the order grand total, no additional payment methods is required. The amount of reward points applied to the order appears with the order totals, subtracted from the grand total, similar to a store credit or gift cards. If reward points are used in conjunction with store credit or a gift card, the reward points are deducted first, and the store credit or gift card is deducted if the order total is greater than the redeemable amount of reward points.
```

File: src/quick-tour/customer-journey.md [hunk 4]
```diff
@@ -2,26 +2,26 @@
...
+Design your site with prepared templates, or create a custom design with features that [invite]({{ site.baseurl }}{% link quick-tour/engage.md %}) people to interact with your store.
 
-## [Increase AOV]({{ site.baseurl }}{% link quick-tour/increase-average-order-value.md %})
+## Increase AOV
 
-Increase average order value with promotions and content that encourage your customers to shop more.
+[Increase average order value]({{ site.baseurl }}{% link quick-tour/increase-average-order-value.md %}) with promotions and content that encourage your customers to shop more.
 
-## [Moment of Purchase]({{ site.baseurl }}{% link quick-tour/moment-of-purchase.md %})
+## Moment of Purchase
```
Comment (shrielenee) [inline src/quick-tour/customer-journey.md]: Change this linked text from "Give" to the "faster and easier way to check out" text.

File: src/quick-tour/customer-journey.md [hunk 5]
```diff
@@ -2,26 +2,26 @@
...
+[Give]({{ site.baseurl }}{% link quick-tour/moment-of-purchase.md %}) your customers a faster and easier way to check out. Calculate shipping and taxes automatically, and integrate multiple payment methods on a single page.
 
-## [Customer Retention]({{ site.baseurl }}{% link quick-tour/customer-retention.md %})
+## Customer Retention
 
-Create and manage newsletters and promotions to keep your customers coming back for more.
+Create and manage [newsletters and promotions]({{ site.baseurl }}{% link quick-tour/customer-retention.md %}) to keep your customers coming back for more.
 
-## [Loyalty &amp; Advocacy]({{ site.baseurl }}{% link quick-tour/loyalty-advocacy.md %})
+## Loyalty &amp; Advocacy
```
Comment (shrielenee) [inline src/quick-tour/customer-journey.md]:
```suggestion
## Loyalty and Advocacy
```

File: src/reports/marketing-reports.md [hunk 6]
```diff
@@ -11,12 +11,12 @@ Marketing reports provide information about the status of shopping carts, the us
...
 The Products in Cart Report lists all products currently in shopping carts, and includes the name and price of each item, the number of carts with the item, and the number of times each item has been ordered.
 
-## [Search Terms]({{ site.baseurl }}{% link marketing/search-terms-report.md %})
+## Search Terms
 
 ![]({{ site.baseurl }}{% link images/images/reports-marketing-search-terms.png %}){: .zoom}
 *Search Terms Report*
 
-The Search Terms Report shows what your customers are looking for in each store view. The report includes the number of matching items found in the catalog, and how many times the search term has been used.
+The хSearch Terms Report]({{ site.baseurl }}{% link marketing/search-terms-report.md %}) shows what your customers are looking for in each store view. The report includes the number of matching items found in the catalog, and how many times the search term has been used.
```
Comment (shrielenee) [inline src/reports/marketing-reports.md]: xSearch?

File: src/sales/checkout-process.md [hunk 7]
```diff
@@ -6,16 +6,16 @@ When the checkout process begins, the transaction shifts to a secure, encrypted
...
+## Step 1: Shipping
 
-The first step of the checkout process is for the customer to complete the shipping address information, and to choose the shipping method.
+The first step of the checkout process is for the customer to complete the [shipping]({{ site.baseurl }}{% link sales/checkout-step1-shipping.md %}) address information, and to choose the shipping method.
 
 ![]({{ site.baseurl }}{% link images/images/storefront-checkout-step1-shipping.png %}){: .zoom}
 _Shipping_
 
-## [Step 2: Review &amp; Payments]({{ site.baseurl }}{% link sales/checkout-step2-payment-review.md %})
+## Step 2: Review &amp; Payments
```
Comment (shrielenee) [inline src/sales/checkout-process.md]:
```suggestion
## Step 2: Review and Payments
```

File: src/sales/sales-menu.md [hunk 8]
```diff
@@ -22,45 +22,45 @@ On the _Admin_ sidebar, click **Sales**.
 ## Menu Options
 
 <!--{% if "Default.B2B Only" contains site.edition %}-->
-### [Quotes]({{ site.baseurl }}{% link sales/quotes.md %})
+### Quotes
 
 Authorized buyers can negotiation the price with the seller by sending a [request]({{ site.baseurl }}{% link sales/quote-price-negotiation.md %}) from the shopping cart.
```
Comment (shrielenee) [inline src/sales/sales-menu.md]: Find a place to link the `({{ site.baseurl }}{% link sales/quotes.md %})` page.

File: src/sales/sales-menu.md [hunk 9]
```diff
@@ -22,45 +22,45 @@ On the _Admin_ sidebar, click **Sales**.
...
+An [invoice]({{ site.baseurl }}{% link sales/invoices.md %}) is a record of the receipt of payment for an order. Multiple invoices can be created for a single order, each with as many, or as few of the purchased products that you specify. Depending on the payment action, payment can be automatically captured when the invoice is generated.
 
-### [Shipments]({{ site.baseurl }}{% link sales/shipments.md %})
+### Shipments
 
-A shipment is a record of the products in an order that have been shipped. As with invoices, multiple shipments can be associated with a single order, until all of the products in the order are shipped.
+A [shipment]({{ site.baseurl }}{% link sales/shipments.md %}) is a record of the products in an order that have been shipped. As with invoices, multiple shipments can be associated with a single order, until all of the products in the order are shipped.
 
-### [Dispatches]({{ site.baseurl }}{% link sales/dispatches.md %})
+### Dispatches
```
Comment (shrielenee) [inline src/sales/sales-menu.md]: Find a place here to link the `({{ site.baseurl }}{% link sales/dispatches.md %})` page

File: src/shipping/carriers.md [hunk 10]
```diff
@@ -16,18 +16,18 @@ The configuration options vary for each carrier. However, all require the follow
...
+[United Parcel Service]({{ site.baseurl }}{% link shipping/ups.md %}) offers domestic and international shipping services by land and air to more than 220 countries.
 
-## [USPS]({{ site.baseurl }}{% link shipping/usps.md %})
+## USPS
 
-The United States Postal Service is the independent postal service of United States government. USPS offers domestic and international shipping services by land and air.
+The [United States Postal Service]({{ site.baseurl }}{% link shipping/usps.md %}) is the independent postal service of United States government. USPS offers domestic and international shipping services by land and air.
 
-## [FedEx]({{ site.baseurl }}{% link shipping/fedex.md %})
+## FedEx
```
Comment (shrielenee) [inline src/shipping/carriers.md]: Add the link in this paragraph that you took from the heading

File: src/shipping/carriers.md [hunk 11]
```diff
@@ -16,18 +16,18 @@ The configuration options vary for each carrier. However, all require the follow
...
 
 Offers domestic and international shipping services by land and air to more than 220 countries.
 
-## [DHL]({{ site.baseurl }}{% link shipping/dhl.md %})
+## DHL
```
Comment (shrielenee) [inline src/shipping/carriers.md]: Add the link in this paragraph that you took from the heading

File: src/shipping/magento-shipping-quick-look.md [hunk 12]
```diff
@@ -2,38 +2,38 @@
...
+Magento Shipping is now available in [seventeen countries]({{ site.baseurl }}{% link shipping/magento-shipping.md %}), where you can create [shipments]({{ site.baseurl }}{% link sales/shipments-create.md %}) with advanced features such as route and package information with dimensional weight, carrier quotes, international details with required export information, harmonization codes, shipping documents and labels, delivery instructions, and tracking information.
 
-## [Locations]({{ site.baseurl }}{% link shipping/magento-shipping-locations.md %})
+## Locations
 
-Configure multiple warehouses, distribution centers, retail stores, drop shippers, and third-party logistics. Allocate shipments to any supported fulfillment location.
+Configure multiple warehouses, distribution centers, retail stores, drop shippers, and third-party logistics. Allocate shipments to any supported fulfillment [location]({{ site.baseurl }}{% link shipping/magento-shipping-locations.md %}).
 
-## [Click &amp; Collect]({{ site.baseurl }}{% link shipping/magento-shipping-locations.md %})
+## Click &amp; Collect
```
Comment (shrielenee) [inline src/shipping/magento-shipping-quick-look.md]:
```suggestion
## Click and Collect
```

File: src/shipping/magento-shipping-quick-look.md [hunk 13]
```diff
@@ -2,38 +2,38 @@
...
+Offer the convenience of ordering online, with convenient customer pickup at a [nearby location]({{ site.baseurl }}{% link shipping/magento-shipping-locations.md %}).
 
-## [Collection Points]({{ site.baseurl }}{% link shipping/magento-shipping-collection-points.md %})
+## Collection Points
 
-Collection points allow customers to order online and have it shipped to a designated collection point, rather than to their address.
+[Collection points]({{ site.baseurl }}{% link shipping/magento-shipping-collection-points.md %}) allow customers to order online and have it shipped to a designated collection point, rather than to their address.
 
-## [Dispatches]({{ site.baseurl }}{% link sales/dispatches.md %})
+## Dispatches
```
Comment (shrielenee) [inline src/shipping/magento-shipping-quick-look.md]: Add the link in this paragraph that you took from the heading

File: src/stores/x-stores.md [hunk 14]
```diff
@@ -5,37 +5,37 @@ title: Stores
...
+Learn to manage the [store]({{ site.baseurl }}{% link stores/stores-all-stores.md %}) hierarchy of websites, stores, and views. Manage taxes, and currency rates, and learn how attributes are used throughout the system. This section also links to a comprehensive configuration reference.
 {: .indent0First}
 
 [![]({{ site.baseurl }}{% link images/images/tile-system.png %})]({{ site.baseurl }}{% link stores/configuration.md %})
 
-## [Configuration]({{ site.baseurl }}{% link stores/configuration.md %})
+## Configuration
 
-Our online Configuration Reference has field descriptions for every setting.
+Our online [Configuration]({{ site.baseurl }}{% link stores/configuration.md %}) Reference has field descriptions for every setting.
```
Comment (shrielenee) [inline src/stores/x-stores.md]: Link should be for "Configuration Reference"

File: src/stores/x-stores.md [hunk 15]
```diff
@@ -5,37 +5,37 @@ title: Stores
...
 {: .indent0First}
 
 [![]({{ site.baseurl }}{% link images/images/tile-taxes.png %})]({{ site.baseurl }}{% link stores/stores-menu.md %})
 
-## [Taxes]({{ site.baseurl }}{% link tax/taxes.md %})
+## Taxes
 
-Learn how to set up tax classes for products and customer groups,and manage tax zones and rates according to the requirements of your locale.
+Learn how to set up [tax]({{ site.baseurl }}{% link tax/taxes.md %}) classes for products and customer groups, and manage tax zones and rates according to the requirements of your locale.
```
Comment (shrielenee) [inline src/stores/x-stores.md]: Remove the comma you added

**Comment by atwixfirster** [comment]
> Thanks for this PR. I've made some comments for revision.
> 
> Also, this PR is entirely too long. I'm keeping it because we need these changes, but in the future limit your PRs to a smaller size.

Hi, @shrielenee !

I do not like a huge info in the single PR too. But in the issue description you wrote `many more in addition to this` and I decided that I should fix all affected pages 😊 

In the next time I will ask you before start work.

I am sorry about this misunderstanding situation from my side.

File: src/catalog/catalog-menu.md [hunk 16]
```diff
@@ -26,7 +26,7 @@ On the Admin sidebar, click <span class="btn"> Catalog </span>.
 
 ## Products
 
-Create new products of every type and manage your inventory.
+Create new [products]({{ site.baseurl }}{% link catalog/products.md %}) of every type, and manage your inventory.
```
[the remark this answers is not in the document]
Comment (atwixfirster) [reply src/catalog/catalog-menu.md]: yep...

**fixed**

File: src/customers/customers-menu.md [hunk 17]
```diff
@@ -40,8 +40,8 @@ Dynamically display content and promotions to specific customers based on proper
 
 <!--{% endif %}-->
 <!--{% if "Default.B2B Only" contains site.edition %}-->
-### [Companies]({{ site.baseurl }}{% link customers/account-companies.md %})
+### Companies
 
-Lists all active company accounts and pending requests, regardless of status setting, and provides the tools needed to create and [manage]({{ site.baseurl }}{% link customers/account-company-manage.md %}) company.
+Lists all active [company accounts]({{ site.baseurl }}{% link customers/account-companies.md %}) and pending requests, regardless of status setting, and provides the tools needed to create and [manage]({{ site.baseurl }}{% link customers/account-company-manage.md %}) company.
```
[the remark this answers is not in the document]
Comment (atwixfirster) [reply src/customers/customers-menu.md]: **fixed**

File: src/marketing/marketing-menu.md (hunk 1 above)
Comment (atwixfirster) [reply src/marketing/marketing-menu.md]: **done**

File: src/marketing/marketing-menu.md (hunk 2 above)
Comment (atwixfirster) [reply src/marketing/marketing-menu.md]: **fixed**

File: src/marketing/rewards-loyalty.md (hunk 3 above)
Comment (atwixfirster) [reply src/marketing/rewards-loyalty.md]: **implemented**

File: src/quick-tour/customer-journey.md (hunk 4 above)
Comment (atwixfirster) [reply src/quick-tour/customer-journey.md]: **done**

File: src/quick-tour/customer-journey.md (hunk 5 above)
Comment (atwixfirster) [reply src/quick-tour/customer-journey.md]: **done**

File: src/reports/marketing-reports.md (hunk 6 above)
Comment (atwixfirster) [reply src/reports/marketing-reports.md]: 😊 

**fixed**

File: src/sales/checkout-process.md (hunk 7 above)
Comment (atwixfirster) [reply src/sales/checkout-process.md]: **done**

File: src/sales/sales-menu.md (hunk 8 above)
Comment (atwixfirster) [reply src/sales/sales-menu.md]: **done**

File: src/sales/sales-menu.md (hunk 9 above)
Comment (atwixfirster) [reply src/sales/sales-menu.md]: **done**

File: src/shipping/carriers.md (hunk 10 above)
Comment (atwixfirster) [reply src/shipping/carriers.md]: **done**

File: src/shipping/carriers.md (hunk 11 above)
Comment (atwixfirster) [reply src/shipping/carriers.md]: **done**

File: src/shipping/magento-shipping-quick-look.md (hunk 12 above)
Comment (atwixfirster) [reply src/shipping/magento-shipping-quick-look.md]: **done**

File: src/shipping/magento-shipping-quick-look.md (hunk 13 above)
Comment (atwixfirster) [reply src/shipping/magento-shipping-quick-look.md]: **done**

File: src/stores/x-stores.md (hunk 14 above)
Comment (atwixfirster) [reply src/stores/x-stores.md]: **done**

File: src/stores/x-stores.md (hunk 15 above)
Comment (atwixfirster) [reply src/stores/x-stores.md]: **removed**

**Comment by atwixfirster** [comment]
@shrielenee , all your suggestions have been implemented.

Could you please review?

Thank you!

File: src/marketing/marketing-menu.md [hunk 18]
```diff
@@ -27,7 +27,7 @@ Create [related product rules]({{ site.baseurl }}{%- link marketing/product-rela
 
 ### Private Sales
 
-[Private Sales]({{ site.baseurl }}{%- link marketing/promotions.md -%}) and other catalog events are a great way to leverage your existing customer base to generate buzz and new leads with exclusive access for members only, or by invitation.
+Private Sales and other catalog events are a great way to leverage your existing customer base to generate buzz and new leads with exclusive access for members only, or by invitation.
```
Comment (shrielenee) [inline src/marketing/marketing-menu.md]: Why did you remove the link here? I just meant make the text "Private Sales" but keep the link. Apologies for the misunderstanding.

Comment (atwixfirster) [reply src/marketing/marketing-menu.md]: > Why did you remove the link here?

Good question.... Probably I am slowdown today...

**fixed**

Please review!

Thank you, @shrielenee

File: src/cms/pages-core.md [hunk 19]
```diff
@@ -4,31 +4,31 @@ title: Core Content
...
+The “[404 Page Not Found]({{ site.baseurl }}{% link cms/pages-default.md %})” page is named for the response code that is returned when a page cannot be found. URL redirects reduce the number of times that this page appears. However, for those times when it is necessary, you might as well take advantage of the opportunity to offer some links to products that the customer might find interesting.
 
 <!--{% if "Default.B2B Only" contains site.edition %}-->
-## [Access Denied]({{ site.baseurl }}{% link customers/account-company-roles-permissions.md %})
+## Access Denied
 
-The “Access Denied” page appears when the permissions that are assigned to a company user prevent access to the page.
+The “[Access Denied]({{ site.baseurl }}{% link customers/account-company-roles-permissions.md %})” page appears when the permissions that are assigned to a company user prevent access to the page.
```
[the remark this answers is not in the document]
Comment (leslietilling) [reply src/cms/pages-core.md]: I do not think it needs to be in quotes. I have seen it in some places and not in others, so we do need some consistency around that.

[the remark this answers is not in the document]
Comment (shrielenee) [reply src/cms/pages-core.md]: @leslietilling I will create another issue for this work.

**Comment by shrielenee** [comment]
running tests
